source/install/self-managed-changelog.md

```diff
@@ -3,13 +3,115 @@
 
 Also see [changelog in progress](https://bit.ly/2nK3cVf) for the next release.
 
-Lastest Mattermost Releases:
+Latest Mattermost Releases:
+- [Release v6.2 - Feature Release](#release-v6.2-feature-release)
 - [Release v6.1 - Feature Release](#release-v6.1-feature-release)
 - [Release v6.0 - Feature Release](#release-v6.0-feature-release)
 - [Release v5.39 - Quality Release](#release-v5-39-quality-release)
-- [Release v5.38 - Feature Release](#release-v5-38-feature-release)
 - [Release v5.37 - Extended Support Release](#release-v5-37-extended-support-release)
 
+## Release v6.2 - [Feature Release](https://docs.mattermost.com/administration/release-definitions.html#feature-release)
+
+**Release Day: 2021-12-16**
+
+### Important Upgrade Notes
+ -  Channel results in the channel autocomplete will include private channels. Customers using Bleve or Elasticsearch for autocomplete will have to reindex their data to get the new results. Since this can take a long time, we suggest disabling autocomplete and running indexing in the background. When this is complete, re-enable autocomplete.
+
+**IMPORTANT:** If you upgrade from a release earlier than v6.1, please read the other [Important Upgrade Notes](https://docs.mattermost.com/upgrade/important-upgrade-notes.html).
+
+### Highlights
+
+#### Playbook Updates
+ - Added Playbooks improvements, including permission schemes, ability to follow runs, and quality of life updates.
+
+#### Boards Updates
+ - Added Boards improvements, including calendar view and @mention autocomplete.
```

Comment:
Propose including a use case for playbooks and board updates.

Reply:
cc @wuwinson @itao For any further details here.

Reply:
Suggested texts with edits:

Playbooks Updates
 - Added more granular control of playbooks access with new Playbooks permission schemes; added the ability to follow playbook runs to stay informed about the procedures you care about; and added other improvements including the ability to search playbooks, share URLs of individual runs and playbooks, and filter runs by playbook.
 
Boards Updates
 - Added a calendar view to stay on track of deadlines and added an @mention autocomplete to mention someone on a card with ease.